sphinx_click/ext.py

```diff
@@ -291,13 +292,28 @@ def _format_command(ctx, show_nested, commands=None):
         yield ''
 
 
+def nested(argument):
+    values = "full", "short", "none"
+    if not argument:
+        return None
+
+    if argument not in values:
+        raise ValueError(
+            'must supply an argument; choose from %s'
```

Comment:
This is a bit misleading. The issue isn't necessarily that they didn't supply an argument. It could be that they had a typo. How about:

    "%r is not a valid value for ':nested:'; allowed values: %s"